### PR TITLE
fheroes2: new port

### PR DESCRIPTION
**Pull request by oleg-derevenetz**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on
macOS 11.2.3 20D91 x86_64
Xcode 12.5 12E262

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ]  referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?
- [x] checked your Portfile with `port lint`?
- [ ]  tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: games/fheroes2/Portfile [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+compiler.cxx_standard   2011
+
+post-destroot {
+    xinstall -d -m 755 ${destroot}${prefix}/share/examples/${name}
+
+    xinstall -m 644 ${worksrcpath}/LICENSE ${destroot}${prefix}/share/doc/${name}
+    xinstall -m 644 ${worksrcpath}/${name}.key ${destroot}${prefix}/share/examples/${name}
+    xinstall -m 755 ${worksrcpath}/script/demo/demo_unix.sh ${destroot}${prefix}/bin
+
```
Comment (reneeotten) [inline games/fheroes2/Portfile:44]: please add the leading `0` in front of the `644` and `755`

**Review by reneeotten — CHANGES_REQUESTED** [review]
a few comments on the Portfile below. I'm not fully clear on how this package works - one still needs to obtain a demo version (or should have bought the original game) before this will run? So what does this port actually do?

**Comment by oleg-derevenetz** [comment]
Hi @reneeotten

> I'm not fully clear on how this package works - one still needs to obtain a demo version (or should have bought the original game) before this will run? So what does this port actually do?

This port installs game engine, which is written from scratch and licensed as GPL-2. But it still needs resource files from original game to run, so user may either download free demo version of original game using bundled script (no campaign, just one map - just to have the overall impression), or purchase original game (it is still available on GOG, for example) and copy resource files to the appropriate place.

**Review by reneeotten — APPROVED** [review]

**Comment by reneeotten** [comment]
thanks @oleg-derevenetz

**Pull request merged**